ducktape/command_line/parse_args.py

```diff
@@ -58,6 +58,12 @@ def create_ducktape_parser():
                         help="Upper bound on number of tests run simultaneously.")
     parser.add_argument("--repeat", action="store", type=int, default=1,
                         help="Use this flag to repeat all discovered tests the given number of times.")
+    parser.add_argument("--subsets", action="store", type=int, default=1,
```

Comment:
We are assuming that all the subsets are taking approximately same execution time. This is a very strong assumption.
I have seen wide skews. To give you an example:
Here 18 tests ran in 36 mins.
https://travis-ci.org/raghavgautam/kafka/jobs/178960665
And here 6 tests ran in 32 mins.
https://travis-ci.org/raghavgautam/kafka/jobs/178960668
Probably it will be better to allow users to run a range of tests say from 1-15, 15-20, 20-45 etc.